src/Content/Changes.php

```diff
@@ -23,86 +27,112 @@
 class Changes
 {
 	/**
-	 * Access helper for the field, in which changes are stored
+	 * Access helper for the cache, in which changes are stored
 	 */
-	public function field(): Field
+	public function cache(): Cache
 	{
-		return $this
-			->site()
-			->version(VersionId::published())
-			->content()
-			->get('changes');
+		return App::instance()->cache('changes');
+	}
+
+	/**
+	 * Returns the cache key for a given model
+	 */
+	public function cacheKey(ModelWithContent $model): string
+	{
+		return match(true) {
+			$model instanceof File => 'files',
+			$model instanceof Page => 'pages',
+			$model instanceof User => 'users'
+		};
 	}
 
 	/**
 	 * Return all files with unsaved changes
 	 */
 	public function files(): Files
 	{
-		return $this->field()->toFiles();
+		$files = new Files([]);
+		$kirby = App::instance();
+
+		foreach ($this->read('files') as $id) {
+			if ($file = $kirby->file($id)) {
+				$files->add($file);
+			}
+		}
+
+		return $files;
 	}
 
 	/**
 	 * Return all pages with unsaved changes
 	 */
 	public function pages(): Pages
 	{
-		return $this->field()->toPages();
+		return App::instance()->site()->find(
+			false,
+			false,
```

Comment:
`Question:` What are these for? `HasChildren::find()` does only accept strings and arrays. And `Collection::find()` also considers all arguments as keys, so why passing twice a `false` key?

Reply:
It might sound a bit stupid, but I really just copied that code over from the field methods. I think it was actually about making sure that we return a collection and not just a single item.

Reply:
:O I never spotted that :D And I guess `false` is casted to and empty string and thus will never actually return a real model.